dot/network/service_test.go

```diff
@@ -295,7 +295,7 @@ func TestBroadcastDuplicateMessage(t *testing.T) {
 	}
 
 	time.Sleep(time.Millisecond * 500)
-	require.Equal(t, 2, len(handler.messages[nodeA.host.id()]))
+	require.Equal(t, 1, len(handler.messages[nodeA.host.id()]))
```

Comment:
Note, this was changed to 1 since we are expecting only one message to have been sent here, see comment on line 291.  I'm not sure why it was working with 2 in development, may be related to changes when `ConnHandler` interface was removed.